SparkPost/2web2ui
# [AC-1972-patchfix] Allowlist user forms

[AC-1972-patchfix] Allowlist user forms

### What Changed

- Allowlisted users forms

### How To Test

- Check if sensitive fields are blocklisted and data for user create and edit form is sent to segment

### To Do
